.coafile

```diff
@@ -29,6 +29,8 @@ bears = SpaceConsistencyBear
 
 [commit]
 bears = GitCommitBear
+host_used = github
+contains_url = True
```

Comment:
This is going to cause gitmate to reject any "multiple commits PR" where each commit does not have an issue linked.  i.e. it is going to reject quite a few of our commits. e.g.  coala/coala@ea48d66b9 coala/coala@49f8884ff coala/coala@49f441bf2 .  coala/coala@39b0a170 .
Are we ready for this?